src/maint/nrepl/impl/docs.clj

```diff
@@ -0,0 +1,99 @@
+(ns nrepl.impl.docs
+  (:refer-clojure :exclude [comparator])
```

Comment:
I think you've copied this here accidentally.

Reply:
yeah, thanks